help/en/package/apps/TabbedPreferences.shtml

```diff
@@ -279,7 +279,7 @@ This will let you to limit what users have the ability to control.
 </dl>
 
 The "Apply" button puts the current settings into effect, without
-saving them permanently.  For a connected throttle to use the
+saving them permanently.  For a connected <a href="../jmri/jmrit/withrottle/UserInterface.shtml">WiThrottle</a> to use the
```

Comment:
We might change the paragraph to explain that clicking the Prefs panel "Save" button (at lower left) will store change settings, but indeed requires any connected (wi)throttles to reconnect. At least a change of Port Number will show up in the WiThrottle pane.